e2e/cypress/e2e/organization/organizations.cy.ts

```diff
@@ -1,4 +1,36 @@
+import { renameOrg } from 'support/api/orgs';
+import { apiAuth } from '../../support/api/apiauth';
+
+const orgPath = `/org`;
+const testOrgNameChange = 'e2erenametest';
+
 describe('organizations', () => {
+  describe('rename', () => {
+    beforeEach(() => {
+      apiAuth().as('api');
+      cy.visit(orgPath);
+    });
+
+    afterEach(() => {
+      apiAuth()
+        .as('api')
+        .then((api) => {
+          renameOrg(api, 'ZITADEL');
+        });
+    });
+
+    it.only('should rename the organization', () => {
+      cy.get('[data-e2e="actions"]').click();
+      cy.get('[data-e2e="rename"]').click();
```

Comment:
when run with `cypress open`, this `data-e2e="rename"`is not always found, do you know what could be the problem @eliobischof 🤔

Reply:
resolved it by initializing the pipe in org-detail.html